source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -247,22 +241,7 @@ void BioXASAppController::setupExporterOptions()
 		AMAppControllerSupport::registerClass<BioXASXASScanConfiguration, AMExporterXDIFormat, AMExporterOptionXDIFormat>(bioXASDefaultXAS->id());
 }
 
-void BioXASAppController::initializeStorageRing()
-{
-	CLSStorageRing::sr1();
-}
-
-void BioXASAppController::initializeBeamline()
-{
-	BioXASBeamline::bioXAS();
-}
-
-void BioXASAppController::initializePeriodicTable()
-{
-	AMPeriodicTable::table();
-}
-
-void BioXASAppController::setupUserInterface()
+void BioXASAppController::setupAcquamanUserInterface()
```

Comment:
Is it important to make a distinction between `setupUserInterface()` and `setupAcquamanUserInterface()`? I can see it being important if there is more than one user interface.